controllers/microsoftUserController.ts

```diff
@@ -86,6 +86,58 @@ router.get(
   }
 );
 
+router.get("/cloneReportings", async (req: Request, res: Response) => {
+  try {
+    const authorisedCloneReportingUsers: string[] = [
+      "d12d68ea-b04f-4bd8-9124-becae7acb9b5",
+      "0d5fd023-dbba-4996-8752-52aa716e83a3",
+      "4853e42e-0927-4daa-987a-a361c83fb6be",
+      "1ffd7271-6822-43e2-a3a8-2bac5839f7a7",
+      "781d5e17-5dff-48da-a84f-c9420c0ed957",
+    ];
+
+    const { fromUserId, toUserId, userId } = req.query;
+    if (userId !== undefined) {
+      if (authorisedCloneReportingUsers.includes(userId.toString()) === false) {
```

Comment:
```suggestion
      if (!authorisedCloneReportingUsers.includes(userId.toString())) {
```

You can change the condition like this.